loader/src/injector/entry.cpp

```diff
@@ -28,8 +28,9 @@ void entry(void *addr, size_t size) {
     solist_drop_so_path(addr, true);
     solist_reset_counters(1, 1);
 
-    struct kernel_version version = parse_kversion();
-    if (version.major > 3 || (version.major == 3 && version.minor >= 8)) {
+    if (struct kernel_version version = parse_kversion();
```

Comment:
This is more confusing. I prefer the old way, also keeping the old indentation for {